applications/system/js_app/examples/apps/Scripts/gui.js

```diff
@@ -49,15 +61,28 @@ eventLoop.subscribe(views.demos.chosen, function (_sub, index, gui, eventLoop, v
     } else if (index === 2) {
         gui.viewDispatcher.switchTo(views.keyboard);
     } else if (index === 3) {
-        gui.viewDispatcher.switchTo(views.longText);
+        gui.viewDispatcher.switchTo(views.bytekb);
     } else if (index === 4) {
+        gui.viewDispatcher.switchTo(views.longText);
+    } else if (index === 5) {
+        let path = filePicker.pickFile("/ext", "*");
```

Comment:
I'm curious whether it's at all possible to integrate the file picker with the rest of the GUI system as an ordinary view and make its API asynchronous. Gonna look into it.